### PR TITLE
some mappings in `n/m/world`

### PR DESCRIPTION
**Pull request by eristhea**
broke my git oops. here's the PR!
some uncontroversial names to begin my journey to 100%ing `world`

### CODE REVIEW

File: mappings/net/minecraft/item/ProjectileItem.mapping [hunk 1]
```diff
@@ -0,0 +1,27 @@
+CLASS net/minecraft/unmapped/C_dokwmwgh net/minecraft/item/ProjectileItem
+	METHOD m_dxnfepcn createEntity (Lnet/minecraft/unmapped/C_cdctfzbn;Lnet/minecraft/unmapped/C_ogbhoqwb;Lnet/minecraft/unmapped/C_sddaxwyk;Lnet/minecraft/unmapped/C_xpuuihxf;)Lnet/minecraft/unmapped/C_ltpsyvhj;
+		ARG 2 pos
+	METHOD m_kzimfaao createDispenserConfig ()Lnet/minecraft/unmapped/C_dokwmwgh$C_jscrvllj;
+	METHOD m_mruwcksw initialiseProjectile (Lnet/minecraft/unmapped/C_ltpsyvhj;DDDFF)V
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/item/ProjectileItem.mapping]:
```suggestion
	METHOD m_mruwcksw initializeProjectile (Lnet/minecraft/unmapped/C_ltpsyvhj;DDDFF)V
```

File: mappings/net/minecraft/item/ProjectileItem.mapping [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+		ARG 1 projectile
+	CLASS C_jscrvllj DispenserConfig
+		FIELD f_gnbrqqah DEFAULT Lnet/minecraft/unmapped/C_dokwmwgh$C_jscrvllj;
+		METHOD m_lfokgmwh builder ()Lnet/minecraft/unmapped/C_dokwmwgh$C_jscrvllj$C_ciqfsiao;
+		CLASS C_ciqfsiao Builder
+			FIELD f_cquruvvg power F
+			FIELD f_gaosolvv uncertainty F
+			FIELD f_okxxxhyo overrideDispenseBehavior Ljava/util/OptionalInt;
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/item/ProjectileItem.mapping]:
```suggestion
			FIELD f_okxxxhyo overrideDispenseEvent Ljava/util/OptionalInt;
```

File: mappings/net/minecraft/item/ProjectileItem.mapping [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+			FIELD f_qqopqovt positionFunction Lnet/minecraft/unmapped/C_dokwmwgh$C_lbgeddve;
+			METHOD m_chenqixq build ()Lnet/minecraft/unmapped/C_dokwmwgh$C_jscrvllj;
+			METHOD m_dqjwnijh positionFunction (Lnet/minecraft/unmapped/C_dokwmwgh$C_lbgeddve;)Lnet/minecraft/unmapped/C_dokwmwgh$C_jscrvllj$C_ciqfsiao;
+				ARG 1 function
+			METHOD m_estpjtyu power (F)Lnet/minecraft/unmapped/C_dokwmwgh$C_jscrvllj$C_ciqfsiao;
+				ARG 1 power
+			METHOD m_nxfppizk uncertainty (F)Lnet/minecraft/unmapped/C_dokwmwgh$C_jscrvllj$C_ciqfsiao;
+				ARG 1 uncertainty
+			METHOD m_somgfudg overrideDispenseBehavior (I)Lnet/minecraft/unmapped/C_dokwmwgh$C_jscrvllj$C_ciqfsiao;
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/item/ProjectileItem.mapping]:
```suggestion
			METHOD m_somgfudg overrideDispenseEvent (I)Lnet/minecraft/unmapped/C_dokwmwgh$C_jscrvllj$C_ciqfsiao;
```

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/item/ProjectileItem.mapping (hunk 3 above)
Comment (eristhea) [reply mappings/net/minecraft/item/ProjectileItem.mapping]: not sure how that happened

**Review by supersaiyansubtlety — APPROVED** [review]

**Review by pyrox0 — APPROVED** [review]
Looks good to me once conflicts are fixed!

**Pull request merged**